frontend/packages/core/src/Resolver/fetch.tsx

```diff
@@ -4,13 +4,21 @@ import _ from "lodash";
 
 import { client, parseErrorMessage } from "../network";
 
-const fetchResourceSchemas = async (type: string): Promise<IClutch.resolver.v1.Schema[]> => {
+const fetchResourceSchemas = async (
```

Comment:
the Schema object will always come from core package, there is no need to override or try both. it's only in `resolveResource` that it's a problem.